website/docs/configuration.md

```diff
@@ -39,19 +39,19 @@ The skills that the Alfred project is using
 
 Default: `npm`
 
-Determine to install dependencies with NPM or Yarn
+Decide whether to install dependencies with NPM or Yarn
 
 ### `configsDir`
 
 Default: `.`
 
-The directory, relative to the project directory, which config files should be written to
+The directory, relative to the project directory, to write config files
```

Comment:
What do you think about this:

```suggestion
The path (relative to project directory) that config files are written to
```

Reply:
I think that looks good! [I committed your suggestion](https://github.com/amilajack/alfred/pull/336/commits/8e8aa9c0292c94df6f4599e0b6122ec7b8f41834).